src/main/java/de/cyklon/reflection/entities/members/impl/ReflectParameterImpl.java

```diff
@@ -61,6 +61,11 @@ public boolean equals(Object obj) {
 		return obj instanceof ReflectParameter<?, ?> rp && rp.getParameter().equals(parameter);
 	}
 
+	@Override
+	public int hashCode() {
+		return parameter.hashCode() ^ method.hashCode();
```

Comment:
Did IntelliJ recommend the xor operator here? I'm nit sure if this is safe and I would rather use `Objects.hash(parameter, method)`

Reply:
no, but this is how hashcodes are created in the overall java implementation